plugins/SegmentEditor/templates/_unprocessedSegmentMessage.twig

```diff
@@ -0,0 +1,18 @@
+{% set visitorLogLinkHtml %}<a href="{{ visitorLogLink }}" target="_blank">{% endset %}
+{% if isSegmentToPreprocess %}
+<p>
+    {{ 'SegmentEditor_UnprocessedSegmentNoData1'|translate('<strong>(' ~ segmentName ~ ')</strong>')|raw }}
```

Comment:
Wondering about the `|raw` here, maybe it could lead to XSS for specially crafted segments, ie. maybe `segmentName` needs to be escaped?